libchannelserver/ChannelRPCServer.h

```diff
@@ -226,6 +244,8 @@ class ChannelRPCServer : public jsonrpc::AbstractServerConnector,
     std::vector<dev::eth::Handler<int64_t>> m_handlers;
 
     dev::stat::ChannelNetworkStatHandler::Ptr m_networkStatHandler;
+    dev::flowlimit::RPCQPSLimiter::Ptr m_qpsLimiter;
+    dev::flowlimit::QPSLimiter::Ptr m_networkBandwidthLimiter;
```

Comment:
if the logic of networkBandwidthLimiter is the same as QPSLimiter then maybe QPSLimiter is not a good name,
if not maybe networkBandwidthLimiter can inherit QPSLimiter

Reply:
Yes, you are right, I think `QPSLimiter` should be renamed to `RateLimiter`.

Reply:
I will update this in the next PR.